src/styles/custom_variables.scss

```diff
@@ -36,26 +36,4 @@ $additional-colors: (
   "grey-40": ($grey-40, $grey-40-inverse),
 );
 
-// navbar related variables
```

Comment:
@datchley These were initial colors added for the boilerplate. We no longer use them, do you see any reason to keep them here?

Reply:
@stubelshe I'd follow your instincts, if it doesn't look right, I'd go ahead and remove them.

Reply:
Yeah, @datchley added these, just wanted his opinion here. My instinct is to yes, remove them.